### PR TITLE
Adding phpstan wrapper for laravel

### PR DESCRIPTION
**Pull request by danielsuguimoto**
Adding larastan, a phpstan wrapper for Laravel by github.com/nunomaduro

### CODE REVIEW

File: resources/tools.json [hunk 1]
```diff
@@ -747,6 +747,19 @@
...
+      "name": "larastan",
+      "summary": "PHPStan extension for Laravel",
+      "website": "https://github.com/nunomaduro/larastan",
+      "command": {
+        "composer-bin-plugin": {
+          "package": "nunomaduro/larastan",
+          "namespace": "phpstan"
+        }
+      },
+      "test": "true",
```
Comment (jakzal) [inline resources/tools.json]: can you verify the plugin is install here please? 

`composer global show nunomaduro/larastan`

Comment (danielsuguimoto) [reply resources/tools.json]: @jakzal it has broken the build test. I'm not sure what happened...in the logs file I can see the plugin being installed, but `composer show` fails

Comment (jakzal) [reply resources/tools.json]: Just realised I made a mistake above. The command should be:

```composer global bin phpstan show ergebnis/phpstan-rules```

I also noticed that many phpstan plugins use `true`. I'll fix that in a separate PR.

Comment (jakzal) [reply resources/tools.json]: Can you remove the tags section? We don't support PHP 7.1 at all.

**Comment by jakzal** [comment]
Thank you

**Pull request merged**